README.md

```diff
@@ -28,8 +28,6 @@ Each instance of Amsterdam Schema exists of:
 2. Metadata about each table in this single dataset;
 3. For each table, a JSON Schema to describe and validate the data in these tables.
 
-An overview of the existing Amsterdam Schemas can be found in the the Amsterdam Schema Registry at https://github.com/Amsterdam/schemas.
```

Comment:
Maybe keep the text and use ref to /datasets ? https://github.com/Amsterdam/amsterdam-schema/tree/master/datasets

Reply:
Done.